sdk/storage/Azure.Storage.Blobs/src/BlobBaseClient.cs

```diff
@@ -2887,6 +2931,7 @@ private void SetNameFieldsIfNull()
                 allowModifications: options?.AllowModifications ?? false,
                 transferValidationOverride: options?.TransferValidation,
                 async: false,
+                false,
```

Comment:
If `DataLakeFileClient.OpenRead()` calls into this `BlobBaseClient.OpenRead()`, we should add upn as a property for `BlobOpenReadOptions`

Reply:
Disregard, it is not possible for customers to access response headers with the OpenRead() API.

Reply:
as discussed offline: OpenRead and ReadTo do not have access to response headers conveniently, we will only add the upn header to Read() since the user can easily access the owner and group response headers.